flow/util/misc.py

```diff
@@ -393,6 +394,28 @@ def parallel_executor(func, iterable, **kwargs):
     return parallel_executor
 
 
+def _deprecated_warning(
+    deprecation: str,
```

Comment:
We could require keyword arguments so that this function is self-documenting at the call site.
```suggestion
    *,
    deprecation: str,
```